### ISSUE TITLE
Incompatible with Cordova 9.0.0

### ISSUE THREAD

**Issue by AngelaRg** [mircoc/cordova-plugin-settings-hook#15]
Getting the following error when trying to build my app:

```
> cordova prepare
[18:01:51]  lint finished in 7.05 s
Using "requireCordovaModule" to load non-cordova module "../cordova/util" is not supported. Instead, add this module to your dependencies and use regular "require" to load it.
[ERROR] An error occurred while running cordova prepare (exit code 1).
```

This plugin is the only one that uses "requireCordovaModule" in my app.

**Comment by mircoc** [comment]
Hi @AngelaRg,

thank you for the bug report!

I should have fixed it with the new release 0.3.1.

Let me know if it works for you now.

Regards

**Comment by AngelaRg** [comment]
Indeed, looks like it was solved in version 0.3.1. Thank you.
